cypress/e2e/calendar/user-permissions-create-calendars.cy.js

```diff
@@ -0,0 +1,75 @@
+import {
+  MultiColumnListCell, PaneHeader, Pane, Button,
+  including, Link
+} from '../../../interactors';
+
+
+import calendarFixtures from '../../fixtures/calendar_e2e_fixtures';
+import permissions from '../../support/dictionary/permissions';
+
+const testServicePoint = calendarFixtures.servicePoint;
+const testCalendar = calendarFixtures.calendar;
+
+
+
+describe('User with Settings (Calendar): Can create and assign new calendars', () => {
+  let testCalendarResponse;
+  before(() => {
+    // login as admin so necessary state can be created
+    cy.loginAsAdmin();
+    // reset db state
+    cy.deleteServicePoint(testServicePoint.id, false);
+
+    // create test service point
+    cy.createServicePoint(testServicePoint, (response) => {
+      testCalendar.assignments = [response.body.id];
+
+      cy.createCalendar(testCalendar, (calResponse) => {
+        testCalendarResponse = calResponse.body;
+      });
+    });
+    cy.logout();
+
+    cy.openCalendarSettings();
+    cy.createTempUser([
+      permissions.calendarCreate.gui,
+    ]).then(userProperties => {
+      cy.login(userProperties.username, userProperties.password);
+      cy.openCalendarSettings();
+    });
+  });
+
+  after(() => {
+    cy.logout();
+
+    // // login as admin to teardown testing data
+    cy.loginAsAdmin();
+    cy.deleteServicePoint(testServicePoint.id, true);
+    cy.deleteCalendar(testCalendarResponse.id);
+  });
+
+
+  it('checks if user can create and assign calendars', () => {
```

Comment:
`it('{test case id from test rail} {test case name from test rail} (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {`